db/institutions.yml

```diff
@@ -855,12 +910,14 @@ ufrankfurt:
 uhamburg:
   aka:
     - University of Hamburg
+    - Hamburg University
+    - UHH
   city: Hamburg
   country: Germany
   departments:
     chyn:
```

Comment:
``chn`` might be better here

Reply:
CHyN is the official abbreviation.. that's why I used this one

Reply:
I see.  I leave this decision to you.  I somehow feel that I would intuit ``chn`` before ``chyn`` but I could see the argument both ways...

Reply:
lets go with the intuitive way then ^^

Reply:
haha, ok!